source/TimeSeries/UnitTests/TimeSeriesBundleToJsonConverterTest.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using Energinet.DataHub.Core.JsonSerialization;
+using Energinet.DataHub.TimeSeries.Application;
+using Energinet.DataHub.TimeSeries.Application.Dtos;
+using Energinet.DataHub.TimeSeries.Application.Enums;
+using Energinet.DataHub.TimeSeries.TestCore.Assets;
+using FluentAssertions;
+using NodaTime;
+using Xunit;
+
+namespace Energinet.DataHub.TimeSeries.UnitTests;
+
+public class TimeSeriesBundleToJsonConverterTest
```

Comment:
nit: I think we usually use plural
```suggestion
public class TimeSeriesBundleToJsonConverterTests
```